### DISCUSSION TITLE
Run task later

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#650] [Help] [answered]
Is there a way to register a schedule that will run 1 minute later. 1 time

**Comment by ghost** [comment]
There's an overload with ``delay`` for ``Scheduler#scheduleTask``
https://github.com/Minestom/Minestom/blob/3afc644c5bf92e15b5744593eef592cc30281300/src/main/java/net/minestom/server/timer/Scheduler.java#L59

**Comment by TheMode** [comment]
See the [wiki](https://wiki.minestom.net/feature/schedulers)

```java
MinecraftServer.getSchedulerManager().scheduleTask(() -> {
    System.out.println("Hey");
}, TaskSchedule.duration(Duration.ofSeconds(1)), TaskSchedule.stop());
```

**Comment by GreatWyrm** [comment, marked as answer]
You would want something like this:
```
MinecraftServer.getSchedulerManager().buildTask(() -> {
            System.out.println("Test");
        }).delay(TaskSchedule.minutes(1)).schedule();
```

**Reply by sqcred** [reply]
thanks
